### PR TITLE
Add grimAuxiliatrix and fix bugs again

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: data/mods/ssb/random-teams.ts [hunk 1]
```diff
@@ -417,7 +423,7 @@ export const ssbSets: SSBSets = {
 export class RandomStaffBrosTeams extends RandomTeams {
 	randomStaffBrosTeam(options: {inBattle?: boolean} = {}) {
 		const team: PokemonSet[] = [];
-		const debug: string[] = []; // Set this to a list of SSB sets to override the normal pool for debugging.
+		const debug: string[] = ['grimAuxiliatrix', 'Darth']; // Set this to a list of SSB sets to override the normal pool for debugging.
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		const debug: string[] = []; // Set this to a list of SSB sets to override the normal pool for debugging.
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -756,6 +755,51 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+				moves.sort((a, b) => a.num! - b.num!);
+				randomMove = this.sample(moves).name;
+			}
+			if (!randomMove) {
+				console.log(moves);
+				return false;
+			}
+
+			this.useMove(randomMove, source);
+			// TODO: need to find a way to cancel a multihit move in effect
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: @KrisXV I require assistance with this part pls ty <3

Comment (KrisXV) [reply data/mods/ssb/moves.ts]: I would check how Substitute handles multihit moves in gen 1 since they interact similarly to that

**Comment by ghost** [comment]
Oh shit that’s actually a good idea. Didn’t even think of that lmao. Thanks!

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -1335,18 +1395,24 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
 		category: "Special",
-		desc: "",
-		shortDesc: "",
+		desc: "User gains +1 speed and changes form. Type depends on form",
+		shortDesc: "User gains +1 speed and changes form. Type depends on form",
 		name: "Unbind",
 		pp: 15,
 		priority: 0,
 		flags: {protect: 1},
 		onTryMove() {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
		onTryMove(pokemon, target, move) {
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -2388,6 +2454,18 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, dance: 1},
 		secondary: null,
+		onTryMove() {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
		onTryMove(pokemon, target, move) {
```

File: data/mods/ssb/abilities.ts [hunk 5]
```diff
@@ -433,6 +433,31 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// grimAuxiliatrix
+	biosteel: {
+		desc: "This Pokemon restores 1/3 of its maximum HP, rounded down, when it switches out and prevents other Pokemon from lowering this Pokemon's stat stages.",
+		shortDesc: "Restores 1/3 of its maximum HP when it switches out. Prevents other Pokemon from lowering its stat stages.",
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts]:
```suggestion
		shortDesc: "This Pokemon restores 1/3 max HP when it switches out. Other Pokemon can't lower this Pokemon's stat stages.",
```

File: data/mods/ssb/moves.ts [hunk 6]
```diff
@@ -1335,18 +1397,24 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		accuracy: 100,
 		basePower: 60,
 		category: "Special",
-		desc: "",
-		shortDesc: "",
+		desc: "User gains +1 speed and changes form. Type depends on form",
+		shortDesc: "User gains +1 speed and changes form. Type depends on form",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		desc: "Has a 100% chance to raise the user's Speed by 1 stage. If the user is a Hoopa in its Confined forme, this move is Psychic type, and Hoopa will change into its Unbound forme. If the user is a Hoopa in its Unbound forme, this move is Dark type, and Hoopa will change into its Confined forme. This move cannot be used successfully unless the user's current form, while considering Transform, is Confined or Unbound Hoopa.",
		shortDesc: "Hoopa: Psychic; Unbound: Dark; 100% +1 Spe. Changes form.",
```
I would also move the speed boost to a secondary effect but meh

File: data/mods/ssb/moves.ts [hunk 7]
```diff
@@ -2388,6 +2456,18 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, dance: 1},
 		secondary: null,
+		onTryMove(pokemon, target, move) {
+			this.attrLastMove('[still]');
+			if (pokemon.species.baseSpecies === 'Meloetta') {
+				return;
+			}
+			this.add('-fail', pokemon, 'move: Unbind');
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
			this.add('-fail', pokemon, 'move: Relic Dance');
```

**Pull request merged**